src/views/Orders.vue

```diff
@@ -3,27 +3,157 @@
         <ion-title>{{ $t("Orders") }}</ion-title>
+        <ion-buttons slot="end">
+          <ion-button>
+            <ion-icon slot="icon-only" :icon="swapVerticalOutline" />
+          </ion-button>
+        </ion-buttons>
+      </ion-toolbar>
+      <ion-toolbar>
+        <ion-searchbar @ionFocus="selectSearchBarText($event)" v-model="queryString" :placeholder="$t('Search')" v-on:keyup.enter="getOrderdetails()"></ion-searchbar>
+      </ion-toolbar>
+      <ion-toolbar>
+        <ion-segment @ionChange="segmentChanged($event)" v-model="segment">
+          <ion-segment-button value="open">
+            <ion-label>{{ $t("OPEN") }}</ion-label>
+          </ion-segment-button>
+          <ion-segment-button value="packedorders">
+            <ion-label>{{ $t("PACKED") }}</ion-label>
+          </ion-segment-button>
+        </ion-segment>
       </ion-toolbar>
     </ion-header>
     
     <ion-content :fullscreen="true">
-   
+      <div v-if="segment == 'open'">
+        <ion-list v-if="products.length > 0">
+          <product-list-item v-for="product in products" :key="product.productId" :product="product"/>
+        </ion-list>
+        <ion-infinite-scroll @ionInfinite="loadMoreOrders($event)" threshold="100px" >
+          <ion-infinite-scroll-content loading-spinner="crescent" :loading-text="$t('Loading')"></ion-infinite-scroll-content>
+        </ion-infinite-scroll>
+      </div>
     </ion-content>
   </ion-page>
 </template>
 
 <script lang="ts">
-import { IonContent, IonHeader, IonPage, IonTitle, IonToolbar } from '@ionic/vue';
-import { defineComponent } from 'vue';
+import { IonButton, IonButtons, IonContent, IonHeader, IonIcon, IonInfiniteScroll, IonInfiniteScrollContent, IonLabel, IonList, IonPage, IonSearchbar, IonSegment, IonSegmentButton,  IonTitle, IonToolbar } from '@ionic/vue';
+import { swapVerticalOutline } from 'ionicons/icons'
+import { defineComponent, ref } from 'vue';
+import { useRouter } from 'vue-router'
+import { useStore } from 'vuex';
+import { mapGetters } from 'vuex';
+import ProductListItem from "@/components/ProductListItem.vue";
+import { translate } from '@/i18n'
+import { showToast } from '@/utils'
 
 export default defineComponent({
   name: 'Orders',
   components: {
+    IonButton,
+    IonButtons, 
     IonContent,
     IonHeader,
+    IonIcon, 
+    IonInfiniteScroll,
+    IonInfiniteScrollContent,
+    IonLabel,
+    IonList,
     IonPage,
+    IonSearchbar,
+    IonSegment,
+    IonSegmentButton,
     IonTitle,
-    IonToolbar
+    IonToolbar,
+    ProductListItem
+  },
+  data (){
+    return {
+      queryString: ''
+    }
+  },
+  methods: {
+    async getOrderdetails (vSize: any, vIndex: any){
+      const viewSize = vSize ? vSize : process.env.VUE_APP_VIEW_SIZE;
+      const viewIndex = vIndex ? vIndex : 0;
+      const payload = {
+      sortBy: 'orderDate',
+      sortOrder: 'Desc',
+      viewSize: viewSize,
+      viewIndex: viewIndex,
+      facilityId: 'STORE_8'
```

Comment:
Hi,

Remove the hardcoded value for facilityId, use the user state for getting the current facilityId.

Reply:
Done Sir!